cmd/preview/create.go

```diff
@@ -46,12 +46,12 @@ func Create(ctx context.Context) *cobra.Command {
 		},
 		Args: func(cmd *cobra.Command, args []string) error {
 			if len(args) != 1 {
-				return errors.New("create namespace requires one argument")
+				return errors.New("'preview destroy' requires one argument")
 			}
 			return nil
 		},
 	}
-	cmd.Flags().StringVarP(&previewType, "type", "t", "personal", "define the preview type that may be created")
+	cmd.Flags().StringVarP(&previewType, "type", "t", "personal", "define the preview environment type. Accepted values are ['personal', 'global']")
```

Comment:
```suggestion
	cmd.Flags().StringVarP(&previewType, "type", "t", "personal", "the type of preview environment to create. Accepted values are ['personal', 'global']")
```

Reply:
what's the different between personal and global? If it's a scope level, not sure `type` is the best word to use. Maybe  `scope`? or `access` ?